packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -223,7 +223,7 @@ class EksClusterStack extends Stack {
       instanceTypes: [
         new ec2.InstanceType('c5.large'),
         new ec2.InstanceType('c5a.large'),
-        new ec2.InstanceType('c5d.large'),
+        new ec2.InstanceType('m7i-flex.large'),
```

Comment:
Were you able to run this integ test and deploy the resource to your account?

Reply:
I was able to run the test case, but I did not try to deploy it.

Reply:
Thank you. Let me know if I can help.